package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "readability",
+  "name": "@mozilla/readability",
```

Comment:
This makes it a [scoped package](https://docs.npmjs.com/misc/scope). We need to publish it as such because there's already a [`readability`](https://www.npmjs.com/package/readability) package on npm. 

Users install scoped packages like any other package, using the full name `@mozilla/readability`. On the publishing side, it only requires a few extra things:

* your npm user needs to be added to the [`mozilla`](https://www.npmjs.com/org/mozilla) organization
* your first publish command needs to be `npm publish --access public` (i.e. explicitly opt into making the package public, which is otherwise the default for unscoped packages).